gammapy/utils/time.py

```diff
@@ -156,3 +160,48 @@ def absolute_time(time_delta, meta):
     """
     time = time_ref_from_dict(meta) + time_delta
     return Time(time.utc.isot)
+
+
+def extract_time_info(row):
+    """Separate the timing metadata from the others inside the event file header
+
+    Parameters
+    ----------
+    row : dict
+        dictionary with all the metadata of an event file
+
+    Returns
+    -------
+    row : dict
+        dictionary with the metadata without the time information
+    time_row : dict
+        dictionary with the time metadata
+    """
+    time_row = {}
+    for name in TIME_KEYWORDS:
+        time_row[name] = row[name]
+        del row[name]
```

Comment:
This line modifies the input dictionary and I think this is not a good pattern. Just create the dictionary with the subset of keywords and leave the inout dict untouched.